Tests/demistomock/demistomock.py

```diff
@@ -408,37 +409,35 @@ def command():
     """(Integration only)
        Retrieves the integration command that is being run
 
-       :return: Integrations command name
-       :rtype: ``str``
+    Returns:
+      str: Integrations command name
+
     """
     return ""
 
 
 def log(msg):
-    """
-    Prints a message to the current incidents war room log
+    """Prints a message to the current incidents war room log
 
-    :type msg: ``str``
-    :param msg: The message to be logged
+    Args:
+      msg (str): The message to be logged
 
-    :return: No data returned
-    :rtype: ``None``
+    Returns:
+      None: No data returned
```

Comment:
no need to add that when no data returned

Reply:
need it so it will be clear that nothing is returned